warp/src/js_exports/constellation.rs

```diff
@@ -30,10 +30,44 @@ impl ConstellationBox {
         }
     }
 
+    pub fn current_size(&self) -> usize {
+        self.inner.current_size()
+    }
+
     pub fn max_size(&self) -> usize {
         self.inner.max_size()
     }
 
+    pub fn select(&mut self, path: &str) -> Result<(), JsError> {
+        self.inner.select(path).map_err(|e| e.into())
+    }
+
+    pub fn set_path(&mut self, path: String) {
+        self.inner.set_path(path.into())
+    }
+
+    pub fn get_path(&self) -> String {
+        self.inner.get_path().to_str().unwrap().into()
```

Comment:
We could use `PathBuf::to_string_lossy` here instead if permitted by wasm32 target.

Reply:
thanks. done.